update privilege management document (#3075)
pingcap/docs: privilege-management.md

| Insert         | `InsertPriv`     | Inserts data to a table             |
| Update         | `UpdatePriv`     | Updates the table data             |
| Delete         | `DeletePriv`     | Deleted the table data             |
| Reload         | `ReloadPriv`     | Executes the `FLUSH` statement       |

> | Config         | `ConfigPriv`     | Dynamicly reload configuration             |

Reviewer: Dynamically*

> | Config         | `ConfigPriv`     | Dynamicly reload configuration             |

Reviewer 2: @wwar Thanks

`SHOW CREATE VIEW` requires the `SHOW VIEW` privilege.

> `SHOW GRANTS` requires the `SELECT` privilege to the `mysql` database. If the target user is current user, `SHOW GRANTS` dosen't require any privilege.

Reviewer 2: ```suggestion
`SHOW GRANTS` requires the `SELECT` privilege to the `mysql` database. If the target user is current user, `SHOW GRANTS` does not require any privilege.
```

Requires the `GRANT` privilege with the privileges granted by `GRANT`.

Requires additional `CREATE USER` privilege when create user implicitly.

`GRANT ROLE` requires `SUPER` privilege.

### REVOKE

> Requires the `GRANT` privilege and those privileges targeted by `REVOKE` statement.

Reviewer 2: ```suggestion
Requires the `GRANT` privilege and those privileges targeted by the `REVOKE` statement.
```

`REVOKE ROLE` requires `SUPER` privilege.

### SET GLOBAL

> Requires `SUPER` privielges.

Reviewer 2: ```suggestion
Requires `SUPER` privilege to set global variables.
```

### ADMIN

> Requires `SUPER` privielges.

Reviewer 2: ```suggestion
Requires `SUPER` privilege.
```

### SET DEFAULT ROLE

> Requires `SUPER` privielges.

Reviewer 2: ```suggestion
Requires `SUPER` privilege.
```

### KILL

> Requires `SUPER` privielges when killing other user connections.

Reviewer 2: ```suggestion
Requires `SUPER` privilege to kill other user sessions.
```
